deepvariant/call_variants.py

```diff
@@ -401,26 +406,30 @@ def call_variants(examples_filename,
 
   # Prepare input stream and estimator.
   tf_dataset = prepare_inputs(source_path=examples_filename, use_tpu=use_tpu)
-  estimator = model.make_estimator(
-      batch_size=batch_size,
-      master=master,
-      use_tpu=use_tpu,
-      session_config=config,
-  )
-
-  # Instantiate the prediction "stream", and select the EMA values from
-  # the model.
-  if checkpoint_path is None:
-    # Unit tests use this branch.
-    predict_hooks = []
+  if FLAGS.use_openvino:
+    ie_estimator = OpenVINOEstimator(checkpoint_path, num_channels_in_checkpoint_model,
```

Comment:
This actually doesn't seem to make sense to me. Where is `num_channels_in_checkpoint_model` from?

Reply:
This variable is defined above, initialized from checkpoint by a common part.
```python
num_channels_in_checkpoint_model = shape_map_for_layers[first_layer][2]
```